pelican/s3.py

```diff
@@ -17,10 +18,12 @@ def s3upload_file(
     :param expiration:
     :return: presigned URL on success, False on failure
     """
+    config = Config(signature_version="s3v4")
```

Comment:
See this thread: https://cdis.slack.com/archives/CLZJVC38B/p1696622326869829?thread_ts=1696605160.215059&cid=CLZJVC38B